### PR TITLE
{math}[fosscuda/2017b] Keras v2.2.0

### PR DESCRIPTION
**Pull request by vanzod**
Depends on:
~#7718~ (Theano)
~#7717~ (PyYAML)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
cermis - Linux debian 9.7, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/fdf2f11b244beea01915f61385a27bff for a full test report.

**Comment by akesandgren** [comment]
Kicking travis

**Comment by akesandgren** [comment]
Travis kicked

**Comment by akesandgren** [comment]
@vanzod did you forget to PR PyYAML with the expected versions?

**Comment by vanzod** [comment]
@akesandgren Sorry for that. I updated the dependencies list in the first comment.

**Comment by vanzod** [comment]
Re-start Travis now that all required PRs have been merged

File: easybuild/easyconfigs/k/Keras/Keras-2.2.0-fosscuda-2017b-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+dependencies = [
+    ('Python', '2.7.14'),
+    ('TensorFlow', '1.8.0', versionsuffix),
+    ('Theano', '1.0.2', versionsuffix),
+    ('h5py', '2.7.1', versionsuffix),
+    ('PyYAML', '3.12', versionsuffix),
+]
+
+use_pip = True
+download_dep_fail = True
```
Comment (akesandgren) [inline easybuild/easyconfigs/k/Keras/Keras-2.2.0-fosscuda-2017b-Python-2.7.14.eb]: Remove download_dep_fail, this is a PythonBundle.

File: easybuild/easyconfigs/k/Keras/Keras-2.2.0-fosscuda-2017b-Python-3.6.3.eb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+dependencies = [
+    ('Python', '3.6.3'),
+    ('TensorFlow', '1.8.0', versionsuffix),
+    ('Theano', '1.0.2', versionsuffix),
+    ('h5py', '2.7.1', versionsuffix),
+    ('PyYAML', '3.12', versionsuffix),
+]
+
+use_pip = True
+download_dep_fail = True
```
Comment (akesandgren) [inline easybuild/easyconfigs/k/Keras/Keras-2.2.0-fosscuda-2017b-Python-3.6.3.eb]: Same here

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 8 out of 8 (7 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/2d5298a45e6258aac8d3acb4474b0508 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @vanzod!

**Pull request merged**